definitions.txt

```diff
@@ -8,6 +8,9 @@
 
 tonne_of_coal_equivalent = 29.308 GJ = tce
 
+# Watt-annual
+watt_annual = watt * year = Wa = wattannual
```

Comment:
I think 'a' is short for Latin _annum_. However, note the precise syntax description at the top of pint's default_en.txt. It's not necessary to have four fields on every line.

In this case, since we just want to enable parsing of e.g. 'kWa', we just want the 'Wa' part to be recognized as 'watt * year'. So it's adequate to simply have the line:
```
Wa = watt * year
```